[2024-10-16T09:54:06.624+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-02-08T00:00:00+00:00 [queued]>
[2024-10-16T09:54:06.642+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-02-08T00:00:00+00:00 [queued]>
[2024-10-16T09:54:06.643+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T09:54:06.644+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2024-10-16T09:54:06.645+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T09:54:06.671+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): insert_rate> on 2024-02-08 00:00:00+00:00
[2024-10-16T09:54:06.678+0000] {standard_task_runner.py:55} INFO - Started process 8041 to run task
[2024-10-16T09:54:06.685+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'exchange_rate_usd_kzt_dag', 'insert_rate', 'scheduled__2024-02-08T00:00:00+00:00', '--job-id', '1480', '--raw', '--subdir', 'DAGS_FOLDER/exchange_rates/__init__.py', '--cfg-path', '/tmp/tmpsxsp0ebw']
[2024-10-16T09:54:06.688+0000] {standard_task_runner.py:83} INFO - Job 1480: Subtask insert_rate
[2024-10-16T09:54:06.806+0000] {task_command.py:389} INFO - Running <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-02-08T00:00:00+00:00 [running]> on host 63ee4fa64e74
[2024-10-16T09:54:06.865+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:205: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2024-10-16T09:54:06.940+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=exchange_rate_usd_kzt_dag
AIRFLOW_CTX_TASK_ID=insert_rate
AIRFLOW_CTX_EXECUTION_DATE=2024-02-08T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-02-08T00:00:00+00:00
[2024-10-16T09:54:06.942+0000] {sql.py:253} INFO - Executing: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'KZT', 447.19983265, '2024-02-08')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;
[2024-10-16T09:54:06.960+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T09:54:07.223+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T09:54:07.236+0000] {sql.py:364} INFO - Running statement: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'KZT', 447.19983265, '2024-02-08')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;, parameters: None
[2024-10-16T09:54:07.239+0000] {sql.py:373} INFO - Rows affected: 1
[2024-10-16T09:54:07.257+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=exchange_rate_usd_kzt_dag, task_id=insert_rate, execution_date=20240208T000000, start_date=20241016T095406, end_date=20241016T095407
[2024-10-16T09:54:07.314+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2024-10-16T09:54:07.344+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-16T10:04:08.724+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-02-08T00:00:00+00:00 [queued]>
[2024-10-16T10:04:08.745+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-02-08T00:00:00+00:00 [queued]>
[2024-10-16T10:04:08.746+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T10:04:08.751+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2024-10-16T10:04:08.758+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T10:04:08.806+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): insert_rate> on 2024-02-08 00:00:00+00:00
[2024-10-16T10:04:08.818+0000] {standard_task_runner.py:55} INFO - Started process 10635 to run task
[2024-10-16T10:04:08.828+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'exchange_rate_usd_kzt_dag', 'insert_rate', 'scheduled__2024-02-08T00:00:00+00:00', '--job-id', '2214', '--raw', '--subdir', 'DAGS_FOLDER/exchange_rates/__init__.py', '--cfg-path', '/tmp/tmp40_3grr3']
[2024-10-16T10:04:08.829+0000] {standard_task_runner.py:83} INFO - Job 2214: Subtask insert_rate
[2024-10-16T10:04:09.012+0000] {task_command.py:389} INFO - Running <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-02-08T00:00:00+00:00 [running]> on host 63ee4fa64e74
[2024-10-16T10:04:09.101+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:205: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2024-10-16T10:04:09.192+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=exchange_rate_usd_kzt_dag
AIRFLOW_CTX_TASK_ID=insert_rate
AIRFLOW_CTX_EXECUTION_DATE=2024-02-08T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-02-08T00:00:00+00:00
[2024-10-16T10:04:09.193+0000] {sql.py:253} INFO - Executing: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'KZT', 447.19983265, '2024-02-08')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;
[2024-10-16T10:04:09.209+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T10:04:09.598+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T10:04:09.610+0000] {sql.py:364} INFO - Running statement: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'KZT', 447.19983265, '2024-02-08')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;, parameters: None
[2024-10-16T10:04:09.614+0000] {sql.py:373} INFO - Rows affected: 1
[2024-10-16T10:04:09.635+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=exchange_rate_usd_kzt_dag, task_id=insert_rate, execution_date=20240208T000000, start_date=20241016T100408, end_date=20241016T100409
[2024-10-16T10:04:09.711+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2024-10-16T10:04:09.745+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
